client/packages/programs/src/api/operations.graphql

```diff
@@ -227,6 +227,11 @@ fragment Encounter on EncounterNode {
   }
   programEnrolment {
     id
+    isImmunisationProgram
+    document {
+      type
+      name
```

Comment:
Fetch enrolment doc info as part of the Encounter query to save having to make an additional query